boards/shields/ap4_evb_disp_shield_rev2/ap4_evb_disp_shield_rev2.overlay

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024, Ambiq Micro Inc. <www.ambiq.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		psram0 = &aps256n;
+	};
+};
+
+&gpio64_95 {
+	status = "okay";
+};
+
+&mspi_aps256n {
+
+	pinctrl-0 = <&mspi0_default>;
+	pinctrl-1 = <&mspi0_sleep>;
+	pinctrl-2 = <&mspi0_psram>;
+	pinctrl-names = "default","sleep","psram";
+	status = "okay";
+
+	ce-gpios = <&gpio32_63 57 GPIO_ACTIVE_LOW>;
+
+	cmdq-buffer-location = ".mspi_buff";
+	cmdq-buffer-size = <256>;
+
+	ambiq,rx-config = <2 0 10 0 1 0 0 0 0 1>;
```

Comment:
Are you sure this is correct? I see in the binding, sfturn is at the last position, now it is third?